templates/vue/cypress/fixtures/user.json

```diff
@@ -1441,239 +1114,6 @@
       "backgroundColor": "#8396a1",
       "textColor": "white",
       "skippable": true,
-      "quiz": [
```

Comment:
Do you not need to add to `typeData` part of this file to indicate activity progress?

Reply:
Hi @wynnset I believe typeData exists on line 1060 in this file

Reply:
@harry02031 sorry I thought maybe you need to add the progress for activity in that typeData, but I do not think it's needed since we are storing that information in a different place now.